guides/security/last_day.md

```diff
@@ -1,10 +1,14 @@
 ## Freelancers/Past Employees
 - [ ] Access to these accounts should be revoked upon completion of the engagement:
   - [ ] GitHub
-  - [ ] LastPass
+  - [ ] 1Password
   - [ ] Google
   - [ ] Slack
   - [ ] Trello
   - [ ] Harvest
+  - [ ] Recruitee
+  - [ ] Airtable Tables
```

Comment:
```suggestion
  - [ ] Airtable
```

Reply:
You can't revoke someones airtable account because we don't have any sort of centralised access, you can only revoke access to individual tables

Reply:
That makes sense. Does revoking access to G Suite automatically revoke access to Airtable?

Reply:
Only if you signed up with Google SSO